.github/workflows/trigger-dag-reuse-wf.yaml

```diff
@@ -45,30 +45,25 @@ jobs:
         run: |
           astro_core_api="https://api.astronomer.io/v1alpha1/organizations/${{secrets.organization_id }}/\
           deployments"
-          tries=60
+          tries=15
```

Comment:
It might make sense to wait for an hour maybe in some scenarios where it might take longer. So tries=30?